packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -220,6 +220,7 @@ export interface Revive<TNodeChange = NodeChangeType>
 		HasChanges<TNodeChange>,
 		CanConflict {
 	type: "Revive";
+	content: ProtoNode[];
 	count: NodeCount;
```

Comment:
We shouldn't need the `count` field now that we do `content.length`.

Reply:
ReturnTo has count but not content so this change is a bit messy since I'd have to split up some of its shared code. I think it's probably cleaner to just keep count, unless you think it's worth it.

Reply:
Let's keep the count for now.